subsys/bluetooth/shell/gatt.c

```diff
@@ -525,21 +525,9 @@ static struct db_stats {
 	size_t ccc_cfg;
 } stats;
 
-struct show_data {
-	const struct shell *shell;
-	struct bt_uuid_16 uuid;
-};
-
 static u8_t print_attr(const struct bt_gatt_attr *attr, void *user_data)
 {
-	struct show_data *data = user_data;
-
-	if (data->uuid.val) {
-		if (!bt_uuid_cmp(&data->uuid.uuid, attr->uuid)) {
-			goto print;
-		}
-		return BT_GATT_ITER_CONTINUE;
-	}
+	struct shell *shell = user_data;
```

Comment:
I think it should be `const struct shell *shell = user_data;`

Reply:
Fixed.